components/cycles/AllCycles.jsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/forbid-prop-types */
+import * as React from 'react';
+import { FlatList, View } from 'react-native';
+import styled from 'styled-components/native';
+import PropTypes from 'prop-types';
+import CycleCard from './CycleCard';
+
+const Title = styled.Text`
+  font-family: 'Montserrat_600SemiBold';
+  font-size: 24px;
+  margin: 15px 5%;
+`;
+
+const AllCycles = (props) => {
+  const { items } = props;
+
+  const renderCard = ({ item }) => (
+    <CycleCard
+      name={item.name}
+      subtext={item.subtext}
+      onIconPress={item.onIconPress}
+      onPress={item.onPress}
+      color={item.color}
+    />
+  );
+
+  return (
+    <View>
+      <Title>Cycles</Title>
+      <FlatList
```

Comment:
Does FlatList allow you to scroll in the case a few more cycles are added?

Reply:
Yep, it'll scroll